### PR TITLE
Move basket page to use catalog service for meta-data

### PR DESCRIPTION
**Pull request by mjfrey**
Make the basket view use catalog data instead of LMS (again).

### CODE REVIEW

**Comment by mjfrey** [comment]
@marjev @vkaracic 
Trying to release this again now that all WL sites are in the catalog.  Can you take a look.

**Review by marjev — APPROVED** [review]

**Comment by marjev** [comment]
LGTM

File: ecommerce/extensions/basket/tests/test_views.py [hunk 1]
```diff
@@ -387,39 +389,50 @@ def test_line_item_discount_data(self):
...
-        course_info_json = json.dumps(course_info)
-        httpretty.register_uri(httpretty.GET, course_url, body=course_info_json, content_type='application/json')
+        course_run_info_json = json.dumps(course_info)
+        course_run_url = '{}course_runs/{}/?partner={}'.format(
+            settings.COURSE_CATALOG_API_URL,
+            self.course.id,
+            self.site.siteconfiguration.partner.short_code
+        )
 
+        httpretty.register_uri(
```
Comment (vkaracic) [inline ecommerce/extensions/basket/tests/test_views.py]: Instead of registering a new URI, can you call `mock_dynamic_catalog_single_course_runs_api`?

**Review by vkaracic — APPROVED** [review]
One comment, otherwise :+1:

File: ecommerce/extensions/basket/tests/test_views.py (hunk 1 above)
Comment (mjfrey) [reply ecommerce/extensions/basket/tests/test_views.py]: I am overwriting the return JSON.  That is why I am not using the built in Mock.

Comment (vkaracic) [reply ecommerce/extensions/basket/tests/test_views.py]: You can save yourself the additional code here by adding an additional keyword argument in that mock. Something like this combination: https://github.com/edx/ecommerce/blob/d4bb441536c25c494d63881ab0841da32139e16a/ecommerce/coupons/tests/mixins.py#L50-L52

**Pull request merged**